internal/server/data/organization.go

```diff
@@ -158,3 +158,28 @@ func UpdateOrganization(tx WriteTxn, org *models.Organization) error {
 func CountOrganizations(tx ReadTxn) (int64, error) {
 	return countRows(tx, organizationsTable{})
 }
+
+type ForgottenDomain struct {
+	OrganizationName   string
+	OrganizationDomain string
+	// LastSeenAt is the last time the user logged into this organization.
+	LastSeenAt time.Time
+}
+
+func GetForgottenDomainsForEmail(tx ReadTxn, email string) ([]ForgottenDomain, error) {
+	var results []ForgottenDomain
+
+	rows, err := tx.Query(`
```

Comment:
The query is definitely easier to read split out vs. a single line, and the generic scanRows() is much nicer too.